ledger/src/ui/src/onboard.h

```diff
@@ -0,0 +1,85 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2021 RSK Labs Ltd
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to
+ * deal in the Software without restriction, including without limitation the
+ * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+ * sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+ * IN THE SOFTWARE.
+ */
+
+#ifndef __ONBOARD
+#define __ONBOARD
+
+#include "os.h"
+#include "os_io_seproxyhal.h"
+#include "pin.h"
+
+// passphrase will be prefixed with "MNEMONIC" from BIP39, the passphrase
+// content shall start @ 8
+unsigned int bolos_ux_mnemonic_from_data(unsigned char *in,
+                                         unsigned int inLength,
+                                         unsigned char *out,
+                                         unsigned int outLength);
```

Comment:
I think we should probably create a header file for `bolos_ux_onboarding_seed_bip39.c` and include that rather than having that header casually lying around here. What do you think?